### PR TITLE
Pick Helper

### PR DESCRIPTION
**Pull request by mattmcmanus**
Hello! Long time listener, first time caller....

## Changes proposed in this pull request
This PR introduces a `pick` helper. When using the {{on}} modifier, I've found myself writing a lot of component actions that grab a particular value off of the emitted event to pass along. This helper makes it easy to pick off a particular value and either return it or send it to an optional action. 

I was considering creating an addon for this, but it fits well within the composable helpers domain, so I figured I'd propose it here first. 

I'm open to suggestions on improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -701,6 +741,27 @@ Returns an array of keys of given object.
 
 **[⬆️ back to top](#table-of-contents)**
 
+#### `pick`
+Receives an object and picks a specified path off of it to pass on. Intended for use with `{{on}}` modifiers placed on form elements.
+
+```hbs
+  <input
+    ...
+    {{on 'input' (pipe (pick 'target.value') this.onInput)}}
```
Comment (knownasilya) [inline README.md:750]: Why have two `pick` helpers here?

**Comment by knownasilya** [comment]
Beautiful!

**Comment by knownasilya** [comment]
@snewcomer any chance this can be merged?

**Comment by lukemelia** [comment]
Looks like a great addition to composable-helpers!

**Comment by snewcomer** [comment]
4.1.0

**Comment by barelyknown** [comment]
😍 _fabulous_ addition to composable-helpers! I went to the `ember-set-helper` repo to see if they had something built in and saw the new `pipe` helper mentioned! Thanks for the nice PR @mattmcmanus.

**Pull request merged**